### PR TITLE
Add render_with/1

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

**Comment by bcardarella** [comment]
Is there something holding up this merge or a wider discussion to be had on its need prior to merging?

**Comment by josevalim** [comment]
I am pretty sure this is good to go but if you can give it a try on a branch and give feedback, it would be immensely helpful. :)

**Comment by bcardarella** [comment]
will do!

**Comment by bcardarella** [comment]
Just following up that this looks good to me. I spoke with @chrismccord yesterday and this moves us forward. Thank you @josevalim

File: guides/server/error-handling.md [hunk 1]
```diff
@@ -74,6 +74,26 @@ During a regular controller request, this exception will be converted to a
 To understand how a LiveView reacts to exceptions, we need to consider two
 scenarios: exceptions on mount and during any event.
 
+## Common scenarios
+
+Sometimes, it is desirable to have error handling in multiple LiveViews. In
+this case, it is possible to check for an `@error` variable in the teplates
```
Comment (alexcastano) [inline guides/server/error-handling.md:80]:
```suggestion
this case, it is possible to check for an `@error` variable in the templates
```

**Pull request merged**